### ISSUE TITLE
tbl_survival level_label not working appropriately? 

### ISSUE THREAD

**Issue by michaelcurry1123** [ddsjoberg/gtsummary#60]
```
library(survival)
library(gtsummary)
library(dplyr)
lung2 <- mutate(lung, s2 = ifelse(sex == 1, "1\2", "2/3"))

tbl_survival(lung2, Surv(time,status)~sex,times = c(500,550)
             ,level_label = "male female"
             )
```
How does this work because when you try to add your own level labels it removes the subheadings? 

![image](https://user-images.githubusercontent.com/30420015/56046963-724db300-5d12-11e9-8776-a00a71836f4f.png)

**Comment by ddsjoberg** [comment]
I made some updates to make it more clear how to use the `level_labels` argument.  I also added an error for when it is not used correctly.

Here's the error:
```
library(survival)
library(gtsummary)
library(dplyr)
lung2 <- mutate(lung, s2 = ifelse(sex == 1, "1\2", "2/3"))

survfit(Surv(time,status)~sex, data = lung2) %>%
  tbl_survival(times = c(500,550), level_label = "male female")
```
![image](https://user-images.githubusercontent.com/26774684/56087160-4544f180-5e33-11e9-9c0d-3738a659366c.png)

And the additional documentation:
![image](https://user-images.githubusercontent.com/26774684/56087167-5a218500-5e33-11e9-9646-18a43fbaa680.png)
![image](https://user-images.githubusercontent.com/26774684/56087172-6e658200-5e33-11e9-8764-b3048ab9af42.png)

Definitely want the function to be easy to use.  If you have suggestions for updates, happy to discuss!
